lib: location: wifi: nRF9160 DK with nRF7002 EK support (#9039)
nrfconnect/sdk-nrf: doc/nrf/releases/release-notes-changelog.rst

   * Sample moved from :file:`samples/nrf9160/nrf_cloud_rest_cell_pos` to :file:`samples/nrf9160/nrf_cloud_rest_cell_location`.

* :ref:`modem_shell_application` sample:

  * Added:

>     * LED #1 (nRF9160 DK)/Purple LED (Thingy:91) indicates 5 seconds that a current location has been acquired by using ''location get'' command.

Reviewer: ```suggestion
    * LED 1 (nRF9160 DK)/Purple LED (Thingy:91) is lit for five seconds indicating that a current location has been acquired by using the ``location get`` command.
```

Author: Changed

>     * Overlays for nRF9160 DK with nRF7002 EK for enabling Wi-Fi scanning support.

Reviewer: ```suggestion
    * Overlay files for nRF9160 DK with nRF7002 EK to enable Wi-Fi scanning support.
```

>       By using this configuration, you can for example trigger of acquiring a current location by using ''location get'' command.

Reviewer: ```suggestion
      With this configuration, you can, for example, acquire the current location by using the ``location get`` command.
```

Author: Changed

* :ref:`location_sample`:

  * Added:

>     * Overlays for nRF9160 DK with nRF7002 EK for enabling of acquiring a current location by using Wi-Fi scanning results.

Reviewer: ```suggestion
    * Overlay files for nRF9160 DK with nRF7002 EK to acquire the current location by using Wi-Fi scanning results.
```

Author: Changed. Although, acquire -> obtain
